api/v1/types/model_types.go

```diff
@@ -50,10 +50,12 @@ type StatsResponse struct {
 	Stats  map[string]ExternalStatBundle `json:"stats"`
 }
 
-// An ExternalEntityListEntry represents the latest CPU and Memory usage of a model entity.
+// An ExternalEntityListEntry represents the latest CPU, Memory & network usage of a model entity.
 // A model entity can be a Pod, a Container, a Namespace or a Node.
 type ExternalEntityListEntry struct {
 	Name     string `json:"name"`
 	CPUUsage uint64 `json:"cpuUsage"`
 	MemUsage uint64 `json:"memUsage"`
+	RxBytes  uint64 `json:"rx_bytes"`
```

Comment:
I'd personally prefer exposing this and filesystem stats as well. Its just that the default resource limits in Kube has to change once these metrics are added. So as long as we can go fix the limits, then adding these metrics is OK by me :)